debian/control

```diff
@@ -30,6 +30,7 @@ Architecture: all
 Depends: python3 (<< 3.7),
          python3 (>= 3.4),
          python3-pkg-resources,
+         python3-distutils | libpython3-stdlib (<< 3.6.5),
```

Comment:
Maybe it would be better to get the latest change from master to get rid of this line?

Reply:
Yes, doing that already as a consequence of re-targeting this for 0.9.3 :)

(I realized the mess it would be to try and make a change for 0.10.0 and push it to a branch that tracks 0.9.x)

Reply:
ah I see. thank you!